src/components/myNotes/NotesList.jsx

```diff
@@ -0,0 +1,82 @@
+import List from "@mui/material/List";
+import Alert from "@mui/material/Alert";
+import { DragDropContext, Droppable, Draggable } from "react-beautiful-dnd";
+import InfiniteScroll from "react-infinite-scroll-component";
+import PropTypes from "prop-types";
+
+import MESSAGES from "config/constants/messages";
+import styles from "pages/styled";
+import changePosition from "utils/changePosition";
+
+import NoteContainer from "./NoteContainer";
+
+const Notes = ({ displayChosenNote, notesToDisplay, setPage, page, shareNote, removeNote }) => (
+  <div style={styles.allNotes}>
+    <DragDropContext
+      onDragEnd={(parameters) => {
+        changePosition(parameters, notesToDisplay);
+      }}
+    >
+      {notesToDisplay.length === 0 ? (
```

Comment:
We can check it in other way, try to avoid checking for 0 number:
```suggestion
      {!notesToDisplay.length && (<firstOption />)}
      {notesToDisplay.length && (<secondOption />)}
```